spec/fixtures/multiline-soa

```diff
@@ -0,0 +1,7 @@
+osuosl.org      IN SOA  ns1.auth.osuosl.org. hostmaster.osuosl.org. (
```

Comment:
1)there is a comment character after the opening parentheses and followed by comment.
> +osuosl.org      IN SOA  ns1.auth.osuosl.org. hostmaster.osuosl.org. ( ;here lies a comment 

2)One with hyphen in the ns server name.
> +osuosl.org      IN SOA  an-ns-with-hyphens1.auth.osuosl.org. hostmaster-with-hyphens.osuosl.org. (